static/scss/partials/fx-bento.scss

```
@@ -255,8 +255,12 @@ body.hide-bento firefox-apps > button {
 }
 
 @media screen and (max-width: 900px) {
-  firefox-apps {
-    margin: 0 var(--bentoPadding);
+  firefox-apps.home-page {
+    margin: 0 0 0 $spacing-lg;
+  }
+
+  firefox-apps.profile-page {
+    margin: 0 $spacing-lg;
```

nit: This specificity may not be necessary. 
```suggestion
  .home-page {
    margin: 0 0 0 $spacing-lg;
  }

  .profile-page {
    margin: 0 $spacing-lg;
```